taskw/warrior.py

```diff
@@ -768,17 +768,15 @@ def task_update(self, task):
         else:
             task_uuid = task['uuid']
 
-        id, original_task = self.get_task(uuid=task_uuid)
-
-        if 'id' in task:
```

Comment:
This was actually a subtle bug -- we were mutating the passed-in `task` object instead of its copy (which we'll generate a few lines below).